lib/active_merchant/billing/gateways/decidir_plus.rb

```diff
@@ -179,8 +179,6 @@ def add_payment_method_id(options)
 
         if options[:debit]
           case options[:card_brand]
-          when 'visa'
-            31
```

Comment:
This kind of structure seems intended for explicitness and reasonability, but NBD